### PR TITLE
Add phpstan and require PHP 8 / Laravel 9

### PR DESCRIPTION
**Pull request by qwerty287**

### CODE REVIEW

File: .github/workflows/CI.yaml [hunk 1]
```diff
@@ -11,22 +11,16 @@ on:
...
-        operating-system: [ubuntu-20.04]
-        php-versions: ['7.4', '8.0', '8.1']
-        dependencies: ['no', 'low', 'beta']
-        exclude:
-          - operating-system: ubuntu-20.04
-            php-versions: '8.1'
-            dependencies: 'low'
-
-    name: PHP ${{ matrix.php-versions }} - ${{ matrix.dependencies }}
+        php-versions: ['8.0', '8.1']
```
Comment (ildyria) [inline .github/workflows/CI.yaml]:
```suggestion
        php-versions: ['8.0', '8.1', '8.2']
```

Comment (ildyria) [reply .github/workflows/CI.yaml]: ;p

**Comment by qwerty287** [comment]
I wouldn't copy `phpstan.neon` from Lychee.

**Comment by ildyria** [comment]
Yeah I will prune the unused part.

**Comment by qwerty287** [comment]
Do you try to fix the issues? It's not a good idea if we both do it :)

**Comment by ildyria** [comment]
> Do you try to fix the issues? It's not a good idea if we both do it :)

Up to you. :)

**Comment by ildyria** [comment]
Or we can merge this one and create a new PR to fix the issues.

**Comment by qwerty287** [comment]
Yeah, that's better I think. (I can create the PR)

**Pull request merged**